policyengine_uk/variables/gov/hmrc/income_tax/allowances/personal_allowance/received_personal_allowance.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_uk.model_api import *
+
+
+class received_personal_allowance(Variable):
```

Comment:
The other allowances are not the personal allowance and we shouldn't group them with it